src/api/sso/jest.config.e2e.js

```diff
@@ -1,14 +1,33 @@
+const { waitForPostgres } = require('@jcoreio/wait-for-postgres');
+
 const baseConfig = require('../../../jest.config.base');
 
-module.exports = {
-  ...baseConfig,
-  rootDir: '../../..',
-  testMatch: ['<rootDir>/src/api/sso/test/e2e/**/*.test.js'],
-  collectCoverageFrom: ['<rootDir>/src/api/sso/src/**/*.js'],
-  // We use jest-playwright to do browser tests, https://github.com/playwright-community/jest-playwright
-  // See the jest-playwright config in the Telescope root ./jest-playwright.config.js
-  preset: 'jest-playwright-preset',
-  // Try reducing the number of tests that can run at once down to 1
-  maxConcurrency: 1,
-  maxWorkers: 1,
+module.exports = async () => {
+  try {
+    console.log('Waiting for Supabase postgres container to become ready...');
+    await waitForPostgres({
+      host: 'localhost',
+      user: 'postgres',
+      password: process.env.POSTGRES_PASSWORD,
+      database: 'postgres',
+      timeout: 60 * 1000,
```

Comment:
It should work in 99% of the time!

Reply:
Oh wow! I wasn't aware we could do something like this in jest configs. 
I'll need to catch up on the recording.